ppr-ui/src/enums/Errors/mhrErrors.ts

```diff
@@ -0,0 +1,5 @@
+export enum MhrErrorMsgs {
+    HAS_MIXED_OWNER_TYPES = 'Cannot have a mix of owner, executor, administrator and bankruptcy trustee roles.',
```

Comment:
These should be in the resources, not enum.

Reply:
Made the changes to resources, please let me know if you have any further feedback regarding this.